src/components/subscriptions/licenses/LicenseManagementTable/bulk-actions/RevokeBulkAction.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+import { RemoveCircle } from '@edx/paragon/icons';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+
+import {
+  useLicenseManagementModalState,
+  licenseManagementModalZeroState as modalZeroState,
+} from '../../LicenseManagementModals/LicenseManagementModalHook';
+import LicenseManagementRevokeModal from '../../LicenseManagementModals/LicenseManagementRevokeModal';
+import { canRevokeLicense, getActiveFilters } from '../../../data/utils';
+import { REVOKED } from '../../../data/constants';
+import { SUBSCRIPTION_TABLE_EVENTS } from '../../../../../eventTracking';
+
+const calculateTotalToRevoke = ({
+  selectedRevocableRows,
+  isEntireTableSelected,
+  activatedUsersCount,
+  assignedUsersCount,
+  revokedUsersCount,
+  tableItemCount,
+  activeFilters,
+}) => {
+  if (!isEntireTableSelected) {
+    return selectedRevocableRows.length;
+  }
+  if (activeFilters.length === 0) {
+    return activatedUsersCount + assignedUsersCount;
+  }
+  const activeStatusFilter = activeFilters.find(filter => filter.name === 'statusBadge');
+  const activeEmailFilter = activeFilters.find(filter => filter.name === 'emailLabel');
+
+  let revokableUsersCount = tableItemCount;
```

Comment:
After further investigation/discussion, we are going to keep this as is. Because the overview counts only change when a email filter is applied (**not** a status filter), always subtracting the revoked users count won't work here, resulting in -1 in some situations that were tested.